test/e2e/ui-properties/randplanetBtn

```diff
@@ -0,0 +1,21 @@
+/**
+ * Created by ruxit on 5/19/16.
+ */
+
+module.exports = {
```

Comment:
Is this a duplicate of the ui-locators.js file? Either way might be better to convert this to a page object to represent the home page (for our apps we always create page objects). You can see the documentation here:  
http://nightwatchjs.org/guide#page-objects

You can then start your tests above as:

'test something': browser => {
  browser.page.home().navigate()
}